docs/src/pages/company/careers/developer-advocate.md

```diff
@@ -0,0 +1,60 @@
+# Developer Advocate
+
+<p class="description">We are looking for a developer advocate to educate on the latest features.</p>
+
+## Details of the Role
+
+- Location: Remote (preference for UTC-5 to UTC+4).
+- Type of work: Contractor (full-time, convertible to an employee position in the future).
+- Start date: Immediately.
+- We're a remote company, we prefer asynchronous communication over meetings.
+- We work independently, the rest of us won't know what you're doing day-to-day unless you tell us.
+
+## About us
+
+See the [careers](/company/careers/) and [about us](https://next.material-ui.com/branding/about/) pages.
+
+### Why we're hiring
+
+Both our open-source community and our premium products are growing fast (x2-3 YoY).
+We need talented people to keep that going!
+
+In our [last developers' survey](/blog/2020-developer-survey-results/#5-how-can-we-improve-material-ui-for-you), we have learned that we can go a long way into allowing developers to be successful with our technology.
+
+### Why this is interesting
+
+Our solution is empowering React developers to build awesome applications. It should be easy, it shouldn't require any advanced technical skills. Hundreds of thousands of developers use Material-UI every month.
+
+## About the role
+
+We are looking for a dedicated educator and experienced developer to join us as a Developer Advocate to help people get familiar with and use Material-UI most effectively.
+
+### What you'll do on a day-to-day basis
+
+- You will compose blog posts. You will be writing tutorials and other technical blog posts.
+- You will participate and Speak at events and online live streams. You will speak about Material-UI related topics on live streams and in-person events.
+- You will create technical videos. You will create video tutorials, demos, and additional videos for the Material-UI YouTube channel.
+- You will build apps, examples, and improve the documentation where needed.
+- You will give feedback to product management, to influence the roadmap of the products based on developers' needs.
+
+## About you
+
+### Skills you should have
+
+- You are fluent in English (both written and spoken).
+- You are a highly empathic person.
+- You are passionate about helping other developers solve problems and have an educational mindset.
+- You have some formal or informal experience in teaching: training, tutoring, mentoring, etc.
+
+### What would be nice if you had, but isn't required
+
+- Experience working with open-source and/or with the open-source community.
+- Experience working with/editing video.
+
+## Compensation
+
+A competitive compensation of up to \$140,000 USD/year, depending on the profile. It could go higher for a person that can significantly push the mission forward. You can find the other perks & benefits in the [careers](/company/careers/#perks-amp-benefits) page.
```

Comment:
```suggestion
Competitive compensation of up to \$140,000 USD/year, depending on the profile. It could go higher for a person that can significantly push the mission forward. You can find the other perks & benefits in the [careers](/company/careers/#perks-amp-benefits) page.
```